Project.toml

```diff
@@ -14,4 +14,4 @@ blis_jll = "6136c539-28a5-5bf0-87cc-b183200dce32"
 
 [compat]
 julia = "1.4"
-blis_jll = "0.8.0"
+blis_jll = "< 0.8.2"
```

Comment:
Thanks for the fix!  Note that the registry won't like this, you should use instead something like
```toml
 blis_jll = "0.8.0-0.8.1"
```
(I hope I got the syntax right).

Out of curiosity, why excluding 0.8.2?

Reply:
Thanks.

There's no 0.8.2 yet. I'm somehow trying to exclude future releases being automatically installed, as it might change the contents of `obj_t`.